### PR TITLE
feat(chromium): roll to Chromium 109.0.5412.0 (r1069273)

### LINKED ISSUE #9233: M109 roll
M109 is going to beta on Thu, Dec 1, 2022 which is around the time we need to try rolling a new version.
TODOs:

1) Update the chrome headless name https://crbug.com/1380881 to reflect the upstream change

cc @jrandolf @Lightning00Blade

### PR DESCRIPTION
**Pull request by Lightning00Blade**
Closes #9233

### CODE REVIEW

File: versions.js [hunk 1]
```diff
@@ -53,7 +54,7 @@ const versionsPerRelease = new Map([
 ]);
 
 // The same major version as the current Chrome Stable per https://chromestatus.com/roadmap.
-const lastMaintainedChromiumVersion = '106.0.5249.0';
+const lastMaintainedChromiumVersion = '108.0.5351.0';
```
Comment (OrKoN) [inline versions.js]: let's specify 107.0.5296.0 as 107.0.5296.0 is relatively new.

File: versions.js [hunk 2]
```diff
@@ -53,7 +54,7 @@ const versionsPerRelease = new Map([
 ]);
 
 // The same major version as the current Chrome Stable per https://chromestatus.com/roadmap.
```
Comment (OrKoN) [inline versions.js]: probably will need a better guideline here, perhaps last 3 major versions?

Comment (Lightning00Blade) [reply versions.js]: How about `Should not be more then 2 major versions behind Chrome Stable (link)` ?

Comment (OrKoN) [reply versions.js]: sg

**Review by OrKoN — APPROVED** [review]

File: test/src/navigation.spec.ts [hunk 3]
```diff
@@ -208,7 +195,7 @@ describe('navigation', function () {
         return (error = error_);
       });
       if (isChrome) {
-        expect(error.message).toContain(getExpectedSSLCertMessage());
+        expect(error.message).toContain(EXPECTED_SSL_CERT_MESSAGE_REGEX);
```
Comment (OrKoN) [inline test/src/navigation.spec.ts]: toMatch?

**Pull request merged**